### PR TITLE
Vagrant box name and mysql optimization

### PR DESCRIPTION
**Pull request by joanhey**

### CODE REVIEW

File: deployment/vagrant/core.rb [hunk 1]
```diff
@@ -29,6 +29,7 @@ def provider_libvirt(config)
 
 def provider_virtualbox(config)
   config.vm.provider :virtualbox do |vb, override|
+    vb.name = "TechEmpower Framework Benchmarks"
```
Comment (nbrady-techempower) [inline deployment/vagrant/core.rb]: Please leave this out. this creates conflict when creating two boxes

Comment (joanhey) [reply deployment/vagrant/core.rb]: OK

File: toolset/setup/docker/databases/mysql/my.cnf [hunk 2]
```diff
@@ -42,6 +42,12 @@ table_definition_cache  = 800
 max_heap_table_size     = 128M
 tmp_table_size          = 128M
 
+#
+# monitoring off
+#
+
+performance-schema = false
```
Comment (msmith-techempower) [inline toolset/setup/docker/databases/mysql/my.cnf]: Can we get some documentation explaining what this does?

Comment (joanhey) [reply toolset/setup/docker/databases/mysql/my.cnf]: https://mariadb.org/maria-10-1-mysql-5-7-commodity-hardware/
https://github.com/TechEmpower/FrameworkBenchmarks/issues/2845#issuecomment-363475071

File: deployment/vagrant/core.rb (hunk 1 above)
Comment (nbrady-techempower) [reply deployment/vagrant/core.rb]: Yeah I understand it is not ideal, but you can rename it after it's up if you'd like, or just create your own vagrant deployment directory

Comment (joanhey) [reply deployment/vagrant/core.rb]: Cancel the first CI test

Comment (msmith-techempower) [reply deployment/vagrant/core.rb]: You can't tell me what to do!

Comment (joanhey) [reply deployment/vagrant/core.rb]: Sorry

File: toolset/setup/docker/databases/mysql/my.cnf (hunk 2 above)
Comment (joanhey) [reply toolset/setup/docker/databases/mysql/my.cnf]: More information: 
https://dev.mysql.com/doc/refman/5.7/en/performance-schema.html
In production is not useful and the performance will be better.

**Comment by msmith-techempower** [comment]
Closing in favor of #3589

**Pull request closed without merging**